### ISSUE TITLE
.firm file

### ISSUE THREAD

**Issue by Ixolaro** [mtheall/ftpd#152]
hello, is there a firm file like this i can acess ftpd directly on boot

**Comment by Ixolaro** [comment]
if there isn't any, can you make one? thank you.

**Comment by Masamune3210** [comment]
ftpd requires the full OS to be running to work, it cant be a firm at least in its current implementation

**Comment by Ixolaro** [comment]
so is trere a .firm bootloader that can boot directly .3dsx files?

**Comment by Masamune3210** [comment]
No, 3dsx requires the homebrew environment which requires the OS to be booted. I think you are slightly misunderstanding the purpose of the file format, a firm file isn't just another homebrew type, it's something that you load either as replacement or before the normal firmware
